docs/UsingNavigators.md

```diff
@@ -103,7 +103,7 @@ A more complete example that demonstrates the pushing and popping of routes coul
 
 ```javascript
 import React, { Component, PropTypes } from 'react';
-import { Navigator, Text, TouchableHighlight, View } from 'react-native';
+import {AppRegistry, Navigator, Text, TouchableHighlight, View } from 'react-native';
```

Comment:
You are right, I used to think the snippet is in the index file

2016-07-25 18:37 GMT+08:00 Alexey <notifications@github.com>:

> In docs/UsingNavigators.md
> <https://github.com/facebook/react-native/pull/9000#discussion_r72042054>:
>
> > @@ -103,7 +103,7 @@ A more complete example that demonstrates the pushing and popping of routes coul
> >
> >  ```javascript
> >  import React, { Component, PropTypes } from 'react';
> > -import { Navigator, Text, TouchableHighlight, View } from 'react-native';
> > +import {AppRegistry, Navigator, Text, TouchableHighlight, View } from 'react-native';
>
> nit: space between { and AppRegistry.
>
> I'd rather add AppRegistry in the places where it's used (it's not used
> in the code snippet where you added it)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/react-native/pull/9000/files/2efe82dffe129143a2cd5147769f064ac8c61d23#r72042054>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKXTLAU4JRqEV2AxzYixXQXglrfnzGJ-ks5qZJIHgaJpZM4JTypP>
> .
>